content/golanguk-conference.md

```diff
@@ -0,0 +1,32 @@
++++
+author = ["Alexandre González"]
+date = "2015-03-27T20:30:00-00:00"
+linktitle = "The Golang UK Conference"
```

Comment:
Do you want it changed on `linktitle` or everywhere? We knew about the `Go|Golang` thing, and we decided to go wtih `Golang` exactly for that reason, I would prefer to don't touch this.